tests/install/openqa_worker.pm

```diff
@@ -17,6 +17,7 @@ EOF
 ');
     wait_still_screen(1);
     my $worker_setup = <<'EOF';
+echo "WORKER_HOSTNAME=127.0.0.1" >> /etc/openqa/workers.ini
```

Comment:
but why do we need this? Shouldn't every sane openSUSE system be able to resolve "localhost" to 127.0.0.1?

Reply:
ok, `localhost` is nicer but I want to consider this in a bit more detail as the openQA-in-openQA tests can also be treated as instructions how systems are setup. What does it mean that we need to add this here? Who else under which circumstances would need this? Does everyone setting up openQA on a virtual machine would suddenly need this as well?

Reply:
That's how 2. would look like: https://github.com/os-autoinst/openQA/pull/4945

Reply:
>     2. Enable the feature only when `WORKER_HOSTNAME = auto` is present in the worker config. It is the same as 1. but we'd avoid an additional setting. Of course it is problematic when the value should actually be "auto".

Do you mean it prevents us to come up with an actual hostname "auto"? I think I can live with that restriction, my host is called `true; DROP TABLE passwords` ;)